aderyn_core/src/ast/literals.rs

```diff
@@ -12,6 +12,7 @@ pub enum LiteralKind {
     String,
     HexString,
     Address,
+    UnicodeString, // notok
```

Comment:
`// notok` ??

Reply:
I copied that whole thing by mistake. (It was from my personal notes)

Reply:
It is okay !